doc/design/restore_existing.md

```diff
@@ -0,0 +1,17 @@
+Restore From Existing Backup
+======
+
+We want to restore an etcd cluster from existing backup that operator has saved before.
+
+We add a field in BackupPolicy:
+```Go
+type BackupPolicy struct {
+    // InitPV...
+    InitPV string `json:"initPV"`
```

Comment:
> operator will create PVC "foo-pvc-xxx", and the PVC binds to PV "foo-pv-xxx". 

How will user know that? The user is not necessarily who owns the operator and able to read the log of the operator.

I think the user input is only the name of the cluster. Ideally they can just specify the cluster name as recovery identification.